CHANGELOG.md

```diff
@@ -90,6 +94,7 @@ if the provided arguments are invalid.
 increased significantly due to modular `AnteHandler` support. Increase GasLimit accordingly.
 * (rest) [\#5336](https://github.com/cosmos/cosmos-sdk/issues/5336) `MsgEditValidator` uses `description` instead of `Description` as a JSON key.
 * (keys) [\#5097](https://github.com/cosmos/cosmos-sdk/pull/5097) Due to the keybase -> keyring transition, keys need to be migrated. See `keys migrate` command for more info.
+* (keys) `--algo` flags and `--hd-path` added to `keys add` command in order to make use of keybase modularization
```

Comment:
In order to determine whether this is a client breaking change, we need to answer this question: does the user need to modify their behaviour to get the command to behave as it used to? What happens if neither `--algo` nor `--hd-path` flags are passed? If the answer is that the command defaults to create same keys as before, then no, this is not a breaking change.

Reply:
In case it is not a breaking change, then this entry should be dropped in the Features section @sunnya97

Reply:
Yeah, by default it still creates the same key.  Ok, will move to features 👍